categories/models.py

```diff
@@ -10,10 +10,24 @@ class Category(TranslatableModel):
         description=models.TextField(verbose_name=_("description"), blank=True),
     )
 
+    principle = models.BooleanField(default=False)
+
     class Meta:
         verbose_name = _("category")
         verbose_name_plural = _("categories")
-        ordering = ("id",)
+        ordering = (
+            "-principle",
+            "translations__name",
+        )
 
     def __str__(self):
-        return self.safe_translation_getter("name", super().__str__())
+        return f'{self.safe_translation_getter("name", super().__str__())}\
```

Comment:
this will result in the following strings, I think ~ `Categoryis principle: False`, which looks kinda weird. would it be better to just add something like `(principal)` after the name, when it is principal and not add it otherwise. i.e. `Cool category` and `Main category (principal)`. or something along these lines.